service/src/main/java/org/mskcc/cmo/metadb/service/impl/MessageHandlingServiceImpl.java

```diff
@@ -41,6 +43,9 @@
     @Value("${metadb.igo_sample_update_topic}")
     private String IGO_SAMPLE_UPDATE_TOPIC;
 
+    @Value("${metadb.dmp.new-sample}")
```

Comment:
no dashes in property names, let's change it to `metadb.dmp_new_sample`

Reply:
👍